src/tests/multihost/alltests/test_services.py

```diff
@@ -213,6 +215,8 @@ def test_0007_bz971435(self, multihost, backupsssdconf):
           2. Should succeed
           3. Should succeed
         """
+        if not multihost.client[0].detect_files_provider():
+            pytest.skip("Files Provider support isn't available, skipping")
```

Comment:
According to the description, this test isn't specific to 'files provider' functionality.

Reply:
'enable_files_domain': 'true' is present in the code